src/main/resources/add-ons-to-index.json

```diff
@@ -4023,6 +4073,7 @@
       "uid": "org.openmrs.module.allergy-ui-module",
       "type": "OMOD",
       "name": "Allergy UI Module",
+      "icon": "user-circle-o",
```

Comment:
Using this icon for this module doesn't make sense. A user icon is a rather common icon which gives us no info about the utility of the module